android/app/src/main/java/com/google/samples/cronet_sample/ViewAdapter.java

```diff
@@ -26,44 +26,38 @@
 import androidx.recyclerview.widget.RecyclerView;
 import com.google.samples.cronet_sample.data.ImageRepository;
 import java.io.ByteArrayOutputStream;
-import java.io.File;
 import java.io.IOException;
 import java.nio.ByteBuffer;
 import java.nio.channels.Channels;
 import java.nio.channels.WritableByteChannel;
 import java.util.concurrent.Executor;
 import java.util.concurrent.Executors;
-import org.chromium.net.CronetEngine;
 import org.chromium.net.CronetException;
 import org.chromium.net.UrlRequest;
 import org.chromium.net.UrlResponseInfo;
 
 public class ViewAdapter extends RecyclerView.Adapter<ViewAdapter.ViewHolder> {
 
     private static final String TAG = "ViewAdapter";
-    private static CronetEngine cronetEngine;
-    private Context context;
+    private final Context context;
 
     public ViewAdapter(Context context) {
```

Comment:
Nit: You won't need to cast further down if you pass the activity straight away.